src/shared/libs/iota/utils.js

```diff
@@ -546,3 +546,52 @@ export const withRequestTimeoutsHandler = (timeout) => {
 
     return handleTimeout;
 };
+
+/**
+ * Parses Conditional Deposit Address Link
+ *
+ * @method parseCDALink
+ *
+ * @param {string} cdaLink
+ *
+ * @returns {object}
+ */
+export const parseCDALink = (input) => {
+
+    const result = {
+        address: undefined,
+        message: undefined,
+        expectedAmount: undefined,
+        timeoutAt: undefined,
+    };
+
+    try {
+        const url = new URL(input, true)
+        const { address, message, amount, timeoutAt } = url.query
+
+        if (address.match(/^[A-Z9]{90}$/)) {
+            result.address = address.toUpperCase();
```

Comment:
```suggestion
            result.address = address;
```

If the address matches the above regex, shouldn't it already be upper case?